topic_folders/communications/tools/slack-and-email.md

```diff
@@ -5,3 +5,30 @@ There are many ways in which you can join our conversations:
 - The [Discuss email list](http://carpentries.topicbox.com/groups/discuss), which community members are welcome to join and post to.
 - Our [other email lists](https://carpentries.org/community/#mailing-lists) are a combination of regional and topic-specific lists.
 - Our [Slack channel](https://swcarpentry.slack.com/), which community members are welcome to [join](https://swc-slack-invite.herokuapp.com/).  Click on "Channels" in the left panel to view all existing channels.
+
+Below are descriptions of selected channels found on TopicBox (as discussed in this [blog post](https://carpentries.org/blog/2020/04/channels-to-join-topicbox/)):
+
+- [**Instructor Trainers**](https://carpentries.topicbox.com/groups/trainers) - A list for topics related to the community of Trainers that prepare new instructors in teaching Data, Software and Library Carpentries lessons. 
+- [**Discuss**](https://carpentries.topicbox.com/groups/discuss) - A group for general discussion of Carpentry teaching practices and community activities and concerns. 
+- [**Discussion Hosts**](https://carpentries.topicbox.com/groups/discussion-hosts) - The community of instructors that hosts community discussion sessions. 
+- [**Instructors**](https://carpentries.topicbox.com/groups/instructors) - An admin-only list where opportunities for badged instructors to teach are announced. 
+- [**Maintainers**](https://carpentries.topicbox.com/groups/maintainers) - A list for topics related to the maintenance of Carpentry lessons. 
+
+**local-*** - A list for discussing The Carpentries in our local and regional communities
+
+- [**Australia and New Zealand**](https://carpentries.topicbox.com/groups/local-aunz)
+- [**Africa**](https://carpentries.topicbox.com/groups/local-africa)
+- [**Bay Area**](https://carpentries.topicbox.com/groups/local-africa)
+- [**Colorado**](https://carpentries.topicbox.com/groups/local-colorado)
+- [**Davis, California**](https://carpentries.topicbox.com/groups/local-davis)
+- [**Germany**](https://carpentries.topicbox.com/groups/local-germany)
+- [**Heidelberg**](https://carpentries.topicbox.com/groups/local-heidelberg)
+- [**Latino America**](https://carpentries.topicbox.com/groups/local-latinoamerica)
+- [**Nordic**](https://carpentries.topicbox.com/groups/local-nordic)
+- [**Seattle**](https://carpentries.topicbox.com/groups/local-seattle)
+- [**Southwestern United States**](https://carpentries.topicbox.com/groups/local-swusa)
+- [**Southeastern United States**](https://carpentries.topicbox.com/groups/local-libcarpentry-southeast-u)
+- [**United Kingdom**](https://carpentries.topicbox.com/groups/local-uk)
+- [**Vancouver, Canada**](https://carpentries.topicbox.com/groups/local-vancouver)
+- [**Washington, DC**](https://carpentries.topicbox.com/groups/local-dc)
+
```

Comment:
suggestion to group regional lists (several countries) , country lists and city lists separately in this section, instead of the alphabetical ordering approach